authorization-common/src/main/java/com/sequenceiq/authorization/utils/LegacyResourceTypes.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.authorization.utils;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public enum LegacyResourceTypes {
```

Comment:
this legacy is related to what? is it related to what @horadla23 removed earlier?
https://github.com/hortonworks/cloudbreak/pull/11513

Reply:
lemme check

Reply:
@topolyai5 I'll remove this part of the code